api.proto

```diff
@@ -107,6 +107,19 @@ message FetchMetadataResponse {
     repeated StreamMetadata metadata = 2; // Information for all streams
 }
 
+// FetchPartitionMetadataRequest is sent to retrieve latest partition metadata.
+// Mainly it is useful to retrieve Highest Watermark and Highest Offset of the partition leader
+message FetchPartitionMetadataRequest {
+    string stream       = 1;
+    int32  partitionID  = 2;
```

Comment:
```suggestion
    int32  partition  = 2;
```

For consistency with other parts of the API.

Reply:
done